spec/models/claim/base_claim_spec.rb

```diff
@@ -74,7 +74,7 @@ class MockSteppableClaim < Claim::BaseClaim
   end
 
   describe '.lgfs_claim_types' do
-    specify { expect(described_class.lgfs_claim_types).to match_array([Claim::LitigatorClaim, Claim::InterimClaim, Claim::TransferClaim]) }
+    specify { expect(described_class.lgfs_claim_types).to match_array([Claim::LitigatorClaim, Claim::InterimClaim, Claim::TransferClaim, Claim::LitigatorHardshipClaim]) }
```

Comment:
just so you are aware i have created this to [deal with the repeated use of these arrays](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3258/files#diff-1b7e18c5d9e6d5f6fad2d9afb07b8968R15-R25). again can be handled before going into master